rmc-docs/src/dashboard.md

```diff
@@ -0,0 +1,11 @@
+# RMC Dashboard
+
+The [RMC Dashboard](./dashboard/index.html) purpose is to show the level of support in RMC for all Rust features.
```

Comment:
Nit: grammar, either "RMC Dashboard's purpose" or "the purpose of..."

Reply:
Please check the new version.